crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -152,11 +168,14 @@ impl<'a> TrackedRenderPass<'a> {
 
     /// Assign a vertex buffer to a slot.
     ///
-    /// Subsequent calls to [`TrackedRenderPass::draw`] and [`TrackedRenderPass::draw_indexed`]
-    /// will use the buffer referenced by `buffer_slice` as one of the source vertex buffer(s).
+    /// Subsequent calls to [`draw`] and [`draw_indexed`] on this
+    /// [`RenderPass`] will use `buffer` as one of the source vertex buffers.
     ///
-    /// The `slot_index` refers to the index of the matching descriptor in
+    /// The `slot` refers to the index of the matching descriptor in
```

Comment:
```suggestion
    /// The `slot_index` refers to the index of the matching descriptor in
```
That's the name of the argument below.